pyveda/training.py

```diff
@@ -343,7 +343,7 @@ def __init__(self, name, mlType="classification", tilesize=[256,256], partition=
         self.percent_cached = percent_cached
         self.sensors = sensors
         self._count = _count
-        self._datapoints = _datapoints
+        self._datapoints = None
         self.id = id
```

Comment:
last thing. `id` is a function in python, so you're assigning that function to `self.id` here.  So let's make the input argument `_id` or `dataset_id` or something like that, so that we don't confuse with any builtin python.